index.html

```diff
@@ -13231,7 +13232,7 @@ <h2>Including Elements in the Accessibility Tree</h2>
             <li>Elements that are focusable, or have an ID <a>attribute</a> and an ancestor with the <pref>aria-activedescendant</pref> attribute that matches the implicit or explicit <a>semantics</a> of the required context role. In either case, the element may receive focus and may fire an <a>accessibility <abbr title="Application Program Interface">API</abbr></a> focus event.</li>
             <li>Elements that have an explicit role or a global <abbr title="Accessible Rich Internet Application">WAI-ARIA</abbr> attribute and do not have <sref>aria-hidden</sref> set to <code>true</code>. (See <a href="#tree_exclusion">Excluding Elements in the Accessibility Tree</a> for additional guidance on <sref>aria-hidden</sref>.)</li>
             <li>Elements that have an ID which is referenced by another element via a <abbr title="Accessible Rich Internet Application">WAI-ARIA</abbr> property and are not <a>hidden</a>.
-            	<p class="note">Text equivalents for hidden referenced objects may still be used in the <a href="https://www.w3.org/TR/accname-aam-1.1/#mapping_additional_nd">name and description calculation</a> [[ACCNAME-1.1]] even when not included in the accessibility tree.</p>
+            	<p class="note">Text equivalents for hidden referenced objects may still be used in the <a href="#mapping_additional_nd" class="accname">name and description calculation</a> even when not included in the accessibility tree.</p>
```

Comment:
Use "computation" for consistency?

```suggestion
            	<p class="note">Text equivalents for hidden referenced objects may still be used in the <a href="#mapping_additional_nd" class="accname">name and description computation</a> even when not included in the accessibility tree.</p>
```